setup.py

```diff
@@ -145,3 +129,23 @@
     setup_requires=['nose', 'mock', 'coverage'],
     test_suite='nose.collector'
 )
+
+
+# check for ttfautohint
+msg = ('Command line tool `ttfautohint` is recommended. Install it with'
+        ' `apt-get install ttfautohint` or `brew install ttfautohint`')
+for p in os.environ.get('PATH').split(':'):
+       os.path.join(p, 'ttfautohint')
+       if os.path.exists(os.path.join(p, 'ttfautohint')):
+         continue
+       else:
+         print(msg)
```

Comment:
it seem that this will print the message several times, even if ttfautohint is found... I'll refactor it before merging.